source/includes/getting-started/_concepts.html.md

```diff
@@ -1,61 +1,67 @@
-# Architecture
+# Background FAQ
 
-There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts. Let's jump right in!
+[//]: # (If any commented statements become false, change prior text)
+
+There are some things you might want to know before you get started. In this section, we're going to walk you through the Bonsai Platform components and some of the basic Bonsai concepts that are frequently asked questions. Let's jump right in!
```

Comment:
The phrase "Bonsai concepts" is very easily confused with "Bonsai Concepts" (as in the concept keyword). Sam previously resolved this ambiguity by using the word "fundamentals"--fundamentals of the Bonsai Platform OR fundamental ideas of the Bonsai Platform